dashboard/agent.py

```diff
@@ -54,6 +54,7 @@ def __init__(
         raylet_name=None,
         logging_params=None,
         disable_metrics_collection: bool = False,
+        agent_id: int = os.getpid(),
```

Comment:
I think we don't need the default id as pid in this case. Can you remove it?

Reply:
Refactored to use a required keyword-only `*` signature.

Reply:
Hmm I might be missing something here, but it still seems to receive os.getpid() as default?

Reply:
Sorry, check now :)